src/nav.config.ts

```diff
@@ -1043,6 +1062,247 @@ const fullNav: FullNav = {
       ],
     },
   ],
+  ['reference/go']: [
+    {
+      links: [
+        {
+          title: 'Getting Started',
+          href: '/reference/go/v0',
+        },
+      ],
+    },
+    {
+      title: 'APIs',
+      links: [
+        {
+          title: 'api()',
+          href: '/reference/go/v0/api/api',
+        },
```

Comment:
I wonder if for this section, if we should add a top level `resources` reference?

So we'd have

resources:
 - NewApi() -> Api
 - NewQueue() -> Queue
 - NewTopic() -> Topic
 - ... etc

Followed by the relevent API level calls and (new resources would also link into these

Api
 - Get()
 - Post()
 - Put()
 - Delete()
 - ... etc ...

Reply:
Yeah I think that's a better way of organising it. I'll make the change now.